tools/ci_build/github/azure-pipelines/py-packaging-pipeline.yml

```diff
@@ -29,6 +29,16 @@ parameters:
   type: boolean
   default: true
 
+- name: enable_olive_windows_cpu
```

Comment:
I don't quite understand your needs.

Do you want to publish the packages publicly? If not, I suggest you create a new yaml file and a new ADO pipeline.